### PR TITLE
Fetch deduplication schemas from schema registry

### PR DESCRIPTION
**Pull request by saulius**
Similarly to morphline transformations and regular event schemas - fetch
deduplication schemas from schema registry service.

@ljank

### CODE REVIEW

File: camus-sweeper/src/main/java/com/linkedin/camus/sweeper/mapreduce/CamusMorphlineAvroKeyJob.java [hunk 1]
```diff
@@ -68,27 +68,28 @@ public void configureJob(String topic, Job job)
...
-        String keySchemaStr = getConfValue(job, topic, "camus.sweeper.avro.key.schema");
-        Schema keySchema;
-        if (keySchemaStr == null || keySchemaStr.isEmpty()) {
-            job.setNumReduceTasks(0);
-            keySchema = inputSchema;
-        } else {
-            keySchema = new Schema.Parser().parse(keySchemaStr);
-        }
+        // checking if we can download a key schema used for deduping. if we don't then we make this a map only
+        // job and set the key schema to the newest input schema
```
Comment (ljank) [inline camus-sweeper/src/main/java/com/linkedin/camus/sweeper/mapreduce/CamusMorphlineAvroKeyJob.java]: falsey comment :)

File: camus-sweeper/src/main/java/com/linkedin/camus/sweeper/mapreduce/CamusMorphlineAvroKeyJob.java [hunk 2]
```diff
@@ -68,27 +68,28 @@ public void configureJob(String topic, Job job)
...
+        try {
+            URI deduplicateSchemaURI = new URI(schemaRegistryHost + "/" + topic + "/latest.deduplication");
```
Comment (ljank) [inline camus-sweeper/src/main/java/com/linkedin/camus/sweeper/mapreduce/CamusMorphlineAvroKeyJob.java]: will latest schema (with more fields) be OK to deduplicate older event (with less fields)?

Comment (saulius) [inline camus-sweeper/src/main/java/com/linkedin/camus/sweeper/mapreduce/CamusMorphlineAvroKeyJob.java]: As long as we will be creating backwards compatible schemas - yes.

**Comment by ljank** [comment]
:+1:

**Pull request merged**